pennylane_lightning_kokkos/_version.py

```diff
@@ -17,4 +17,4 @@
 """
 
 
-__version__ = "0.30.0-dev1"
+__version__ = "0.30.0rc0"
```

Comment:
Probably not important, but we usually do:
```suggestion
__version__ = "0.30.0-rc0"
```

Reply:
Docs allow either, though no period/dash is reported often more than if it is included. Either way, I don't mind.